### PR TITLE
Upgrade to new tc-platform version (stripe name support in DC and M&M)

### PR DESCRIPTION
**Pull request by mathieucarbou**
### TODO
- [x] Merge PR https://github.com/Terracotta-OSS/terracotta-platform/pull/828/
- ~~[ ] Merge PR https://github.com/Terracotta-OSS/terracotta-platform/pull/757~~
- [x] Release tc-platform

### CODE REVIEW

**Comment by mathieucarbou** [comment]
`5.8.0-pre21` upload is in progress

**Comment by mathieucarbou** [comment]
closing - need to do another release to add stripe name to M&M support also.

File: gradle.properties [hunk 1]
```diff
@@ -10,11 +10,11 @@ sizeofVersion = 0.3.0
 jaxbVersion = 2.3.1
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.8.0-pre20
+terracottaPlatformVersion = 5.8.0-pre22
```
Comment (mathieucarbou) [inline gradle.properties:13]: this is not a mistake. next is 22.

File: gradle.properties [hunk 2]
```diff
@@ -10,11 +10,11 @@ sizeofVersion = 0.3.0
...
 terracottaApisVersion = 1.7.0-pre5
```
Comment (saurabhagas) [inline gradle.properties:14]: @mathieucarbou can you also bump the following dependencies:
terracottaApisVersion = 1.7.0-pre6
terracottaCoreVersion = 5.7.0-pre34
terracottaPassthroughTestingVersion = 1.7.0-pre16

**Pull request merged**